client/src/connection/rest/SASContentAdapter.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright © 2024, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { FileType, Uri, authentication } from "vscode";
```

Comment:
This file is an adapter to implement on REST service. This feature looks not necessarily specific to REST.
UI feature should be in UI component
https://github.com/sassoftware/vscode-sas-extension/blob/main/client/src/components/ContentNavigator/index.ts#L105
You can see warning message there. Please follow the pattern to make UI strings translatable.